packages/core/admin/admin/src/content-manager/pages/EditView/components/DocumentActions.tsx

```diff
@@ -29,14 +47,18 @@ interface DocumentActionDescription {
    */
   position?: 'panel' | 'header';
   dialog?: DialogOptions | NotificationOptions | ModalOptions;
```

Comment:
Would something like this be possible to avoid `dialog.dialog` and
`dialog.modal` etc
```suggestion
  dialog?: DialogOptions 
  notification?:  NotificationOptions 
  modal?:  ModalOptions;

```